src/NUnitFramework/testdata/AssertMultipleData.cs

```diff
@@ -260,6 +265,88 @@ public void AssumptionInBlock()
                 Assume.That(2 + 2 == 4);
             });
         }
+
+#if ASYNC
+        [Test]
+        public void ThreeAssertsSucceed_Async()
+        {
+            Assert.Multiple(async () =>
+            {
+                await Task.Delay(1000);
```

Comment:
Could be shorter. I added delays in most cases as a way to use await, so that the lambda actually runs asynchronously. In a few cases, some delay was needed or the test would never have failed in the first place. I'll go through them all and see what I can do.